src/pages/home/HeaderView.js

```diff
@@ -172,12 +173,14 @@ function HeaderView(props) {
     const shouldShowBorderBottom = !isTaskReport || !props.isSmallScreenWidth;
     const shouldDisableDetailPage = ReportUtils.shouldDisableDetailPage(props.report);
 
+    const isLoading = !props.report || !title;
+
     return (
         <View
             style={[styles.appContentHeader, shouldShowBorderBottom && styles.borderBottom]}
             dataSet={{dragArea: true}}
         >
-            <View style={[styles.appContentHeaderTitle, !props.isSmallScreenWidth && styles.pl5]}>
+            <View style={[styles.appContentHeaderTitle, !props.isSmallScreenWidth && !isLoading && styles.pl5]}>
                 {props.isSmallScreenWidth && (
```

Comment:
```suggestion
                {!isLoading && props.isSmallScreenWidth && (
```
<img width="321" alt="Screenshot 2023-11-16 at 3 42 04 PM" src="https://github.com/Expensify/App/assets/108292595/ee6e6323-3b27-4bac-baef-ad75b0a41b28">